python/lsst/ts/Dome/dome_csc.py

```diff
@@ -3,6 +3,8 @@
 import asyncio
 import logging
 import pathlib
+import math
+import numpy as np
```

Comment:
Please insert a blank line before the 3rd party imports

Reply:
Thanks, you made this comment before. I am still getting used to this. Done.